Incorrect area and location id for Service hook subscriptions

helaili opened the pull request:
Fixes #123 

Area and location id for service hooks subscriptions don't match with what you get with an `OPTIONS` query to `https://xxxxx.visualstudio.com/DefaultCollection/_apis`

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/Microsoft/vsts-node-api?pullRequest=125) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/Microsoft/vsts-node-api?pullRequest=125) before we can accept your contribution.<br/><br/>:x: helaili [sign now](https://cla.opensource.microsoft.com/Microsoft/vsts-node-api?pullRequest=125)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/Microsoft/vsts-node-api?pullRequest=125) it.</sub>

helaili commented:
@bryanmacfarlane as the original contributor of the lines I have changed, could you please review this PR? 🙏

stephenmichaelf commented:
@helaili Not sure why the apis.json file was added? And there are some changes that add your username.

Biggest question though is the changes to the API file. These are auto generated files. What is driving this change? I can look into regen if there are issues, this one hasn't been updated in a while.

Also worth nothing, this API should only be used for on prem, it's a deployment level API.

helaili commented:
Sorry @stephenmichaelf the original PR got polluted by some later work, not too sure how that happened. I probably pushed to the wrong remote. I must admit I have moved one from this and implemented my own access layer int the meantime.
